styles/globals.css

```diff
@@ -56,6 +56,10 @@ body {
   color: rgb(var(--color-text-base));
 }
 
+ul{
+  list-style: unset !important;
```

Comment:
try to avoid usage of !important not a good CSS practice

Reply:
Agree but here I am overriding the default styling of tailwind, that's why have to use it